src/guess-number.app.js

```diff
@@ -0,0 +1,60 @@
+// What do I need to import from other js files?
+import compareNumbers from './guess-number.js';
+
+
+
+
+
```

Comment:
You have a lot of unnecessary white space. I took a point off for "Project and code files are: indented/spaced correctly/clean" because of that, but if you delete the whitespace and resubmit I'll give you the point back.